templates/shared/actions/runner_badge.tmpl

```diff
@@ -0,0 +1,26 @@
+{{$ := .Badge}}
```

Comment:
And I would suggest to always the `.Badge.Xxx`, to avoid unnecessary key conflicts in the future.

For example, what if people would like to add some "options" beside the "Badge"?